modules/sdk-core/src/bitgo/tss/ecdsa/ecdsa.ts

```diff
@@ -477,7 +572,29 @@ export function convertAShare(share: AShare): SignatureShareRecord {
   return {
     to: getParticipantFromIndex(share.i),
     from: getParticipantFromIndex(share.j),
-    share: `${share.k}${delimeter}${share.alpha}${delimeter}${share.mu}${delimeter}${share.n}`,
+    share: `${share.k}${delimeter}${share.alpha}${delimeter}${share.mu}${delimeter}${share.n}${delimeter}${
+      share.ntilde
+    }${delimeter}${share.h1}${delimeter}${share.h2}${delimeter}${share.proof?.z || ''}${delimeter}${
+      share.proof?.u || ''
+    }${delimeter}${share.proof?.w || ''}${delimeter}${share.proof?.s || ''}${delimeter}${
+      share.proof?.s1 || ''
```

Comment:
Fixing this in my branch, but looks like a bug here right? `share.proof.s1` is there twice?